google/ggrc-core
# GGRC-6547 Incorrect setup of Verified date for Assessment

# Issue description

Verified date should be readonly

# Steps to test the changes

Steps to reproduce:
1. Open Audit page
2. Create Assessment (asmt should be in 'Not Started' state)
3. Export Assessment
4. Add a verifier, set 'In Review' state, update verified date (e.g. set previous date - 1/7/2019)
5. Import Assessment
6. Open Assessment in UI and look at the verified date: is displayed

# Solution description

Validate that we do not update Verified date during import Assessment

# Sanity checklist




# PR Review checklist
